CHANGELOG.md

```diff
@@ -6,10 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-
 ### Added
 - Support for setting file capabilities via the RPMTAGS_FILECAPS header.
 - `PackageMetadata::get_file_entries` method can get capability headers for each file.
+- Support for symbolic link in file mode.
+- Make file type const `REGULAR_FILE_TYPE` `DIR_FILE_TYPE` `SYMBOLIC_LINK_FILE_TYPE` public, because `FileMode::file_type` is public, sometimes we need this const to determin file type.
```

Comment:
Typo "determin"

Reply:
Oh, my bad, will keep an eye for this kind of issue if have another pr, thanks.